docs/src/pages/demos/pickers/MaterialUIPickers.tsx

```diff
@@ -0,0 +1,60 @@
+import 'date-fns';
+import React from 'react';
+import PropTypes from 'prop-types';
+import Grid from '@material-ui/core/Grid';
+import { createStyles, withStyles, WithStyles } from '@material-ui/core/styles';
+import DateFnsUtils from '@date-io/date-fns';
+import { MuiPickersUtilsProvider, TimePicker, DatePicker } from 'material-ui-pickers';
+
+const styles = createStyles({
+  grid: {
+    width: '60%',
+  },
+});
+
+export type Props = WithStyles<typeof styles>;
+
+export interface State {
+  selectedDate: any;
```

Comment:
Why was the `any` necessary here? Is there maybe some fix we can propose upstream to `material-ui-pickers`?

Reply:
@eps1lon Looks like their typing for date is really just an alias for `any`

material-ui-pickers date typing file:
[`date.ts`](https://github.com/dmtrKovalenko/material-ui-pickers/blob/develop/lib/src/typings/date.ts)

Created issue dmtrKovalenko/material-ui-pickers#970

Reply:
I will keep an eye on it. I think we should use an actual type here. If we use the correct one an update in material-ui-pickers is fine. If not we get notified with a test failure on update.

Reply:
Please fill free to use Date type here. 
In pickers we can provide only Moment | Luxon | Date | Dayjs. That’s a smth that we will change in v3. But you are using date-fns utility, so you will never get the other types in response.

Reply:
@eps1lon agreed. I made PR #15144 to use the `Date` type in this demo.

@dmtrKovalenko thank you for your help!